mitodl/hugo-course-publisher
# use thumbnail sized image on mobile

#### Pre-Flight checklist

#### What are the relevant tickets?
https://trello.com/c/qiCvN0U8/90-add-thumbnail-to-mobile-course

#### What's this PR do?
This pr updates the course to display the image  from `course_thumbnail_image_url` in the markdown when the course is viewed on a medium or smaller screen 

#### How should this be manually tested?
Run the course publisher. Verify that the thumbnail image is used in the mobile view
